src/pages/home/ReportScreen.js

```diff
@@ -272,7 +272,7 @@ class ReportScreen extends React.Component {
                         ) : (
                             <HeaderView
                                 reportID={reportID}
-                                onNavigationMenuButtonClicked={() => Navigation.goBack(ROUTES.HOME)}
+                                onNavigationMenuButtonClicked={() => Navigation.goBack(ROUTES.HOME, false, true)}
```

Comment:
The `ReportScreen` may return `HeaderView` or `MoneyRequestHeader`. We need to apply same logic for `MoneyRequestHeader`
https://github.com/Expensify/App/blob/820fa93ad1e799c12d8b639b98018b5350ea17af/src/components/MoneyRequestHeader.js#L112